requirements-dev.txt

```diff
@@ -10,3 +10,4 @@ pytest-django==4.8.0
 pytest==8.3.5
 pytest-cov==6.1.1
 coverage==7.7.0
+django-extensions==4.1
```

Comment:
Let's remove this dependency.
I'm updating the project to Django 5.2, which comes with the auto-import of models in Django shell, which is what I believe the reason you added this dependency.